### PR TITLE
Add search with Cloutavista

### PR DESCRIPTION
**Pull request by daniel-keyes**
Hi 👋 

Very excited to be submitting my first PR! 

This is a simple integration that offers a redirect for Cloutavista search at the bottom of the search bar.
We have an API ready so in the future, we can support an integrated results page within BitClout.com

You can see a live example on this Node: http://147.182.191.16/

Desktop view:
Desktop Cloutavista Search

Mobile view:
Mobile Cloutavista Search

Looking forward to all comments and suggestions. Thanks!

### CODE REVIEW

**Comment by tijno** [comment]
nice - what improvements over the current search does it offer?

Note - this should probably follow CIP @daniel-keyes details here: http://github.com/bitclout/cips/

**Comment by diamondhands0** [comment]
This is a small enough change that I don't think we need a CIP for it (I would put it in the Medium category). It was discussed publicly [here](https://bitclout.com/posts/19e4425fd705c166408ce13077e8386cddd32b4dca4ad6bbe9d375722b027490?feedTab=Following) and they got buy-in from the community and core dev team so I think it'll be good to merge once I and another core dev finishes reviewing it.

@daniel-keyes, for the more native UI changes you were telling me about, can you just open a Github Discussion for them, as described [here](https://bitclout.com/posts/19e4425fd705c166408ce13077e8386cddd32b4dca4ad6bbe9d375722b027490?feedTab=Following) for Medium changes? It would be good to have more community input for that since it's a bigger change that will affect all node operators.

**Comment by tijno** [comment]
ok all good. I had not seen the specific comment about the link.

all makes sense now.

File: src/styles.scss
Comment (lazynina) [inline src/styles.scss:25]: Please revert whitespace changes here - it's hard to tell what has really changed.

File: src/app/search-bar/search-bar.component.html
Comment (lazynina) [inline src/app/search-bar/search-bar.component.html:32]: With the addition of the new element at the bottom, the last creator that was hovered over is still being displayed as selected. Let's add a mouseout handler.


In search-bar.component.ts
```suggestion
          (mouseover)="_handleMouseOver(creator.Username, ii)"  (mouseout)="_handleMouseOut(creator.Username, ii)">
```
```
  _handleMouseOut(creator: string, index: number) {
    if (this.creatorSelected === creator) {
      this.creatorSelected = "";
    }
    if (this.selectedCreatorIndex === index) {
      this.selectedCreatorIndex = -1;
    }
  }
```

File: src/app/search-bar/search-bar.component.html
Comment (lazynina) [inline src/app/search-bar/search-bar.component.html:58]:
```suggestion
        <a target="_blank" class="link--unstyled fs-14px text-grey7"
```

File: src/app/search-bar/search-bar.component.html
Comment (lazynina) [inline src/app/search-bar/search-bar.component.html:60]:
```suggestion
          <i class="fas fa-external-link-alt" aria-hidden="true"></i>
```

File: src/app/search-bar/search-bar.component.html
Comment (lazynina) [reply src/app/search-bar/search-bar.component.html:58]: When we hover over this element, it should highlight the background like it does for the above elements in the search bar.

**Review by lazynina — CHANGES_REQUESTED** [review]
There are a bunch of whitespace/alignment changes - please revert these so this PR only adds the new element to the search bar.

Additionally, see feedback on the hover actions within the search bar.

File: src/styles.scss
Comment (daniel-keyes) [reply src/styles.scss:25]: ah, will fix this asap. Thanks

File: src/app/search-bar/search-bar.component.html
Comment (daniel-keyes) [reply src/app/search-bar/search-bar.component.html:32]: Good catch! At the moment, this happens for all results in the popdown. We can add a mouseout for each one so the result never stays highlighted when you leave that specific row.

**Comment by brootle** [comment]
@diamondhands0 hey, can you finally give me blue verified mark? I did try to contact via node.admin@bitclout.com but not sure if that even works, thanks!

**Comment by daniel-keyes** [comment]
@lazynina made all changes per your suggestions. You can see the live version here: http://147.182.191.16/

**Comment by lazynina** [comment]
@daniel-keyes - The content of your change looks good, but there are still a ton of spacing changes that need to be removed.  Additionally, please resolve merge conflicts in `src/stytes.scss`

**Comment by daniel-keyes** [comment]
@lazynina merge conflicts have been solved. I fixed the spacing changes that seemed to have happened from auto-reformatting. Github diff still shows changes for rows that no changes were made to. I think this might have to do with whitespace, but when doing diff locally, I only see the specific rows where code was added. 

Any idea what this is or how to fix it?

**Comment by brootle** [comment]
> @lazynina merge conflicts have been solved. I fixed the spacing changes that seemed to have happened from auto-reformatting. Github diff still shows changes for rows that no changes were made to. I think this might have to do with whitespace, but when doing diff locally, I only see the specific rows where code was added.
> 
> Any idea what this is or how to fix it?

@daniel-keyes  disable Prettier plugin in VS Code, go to github main and copy original raw file from there, after that add your changes, worked for me when I had similar problem

**Comment by daniel-keyes** [comment]
@lazynina all whitespace issues should be fixed now. Thanks @brootle!

**Review by lazynina — APPROVED** [review]
Great works! And thanks @brootle on the prettier config response.

**Pull request merged**
